examples/activate-mongodb-atlas-resources.template.yaml

```diff
@@ -0,0 +1,341 @@
+#    Type: AWS::CloudFormation::TypeActivation
+#    Properties:
+#      PublicTypeArn: !Join [ "", [ 'arn:aws:cloudformation:',!Ref "Region",'::type/resource/bb989456c78c398a858fef18f2ca1bfc1fbba082/MongoDB-Atlas-NetworkPeering' ] ]
+#      Type: RESOURCE
+#      TypeName: MongoDB::Atlas::NetworkPeering
+#      ExecutionRoleArn: !GetAtt MongoDBCustomResourceExecutionRole.Arn
+#  ActivatePrivateEndpointType:
+#    Type: AWS::CloudFormation::TypeActivation
+#    Properties:
+#      PublicTypeArn: !Join [ "", [ 'arn:aws:cloudformation:',!Ref "Region",'::type/resource/bb989456c78c398a858fef18f2ca1bfc1fbba082/MongoDB-Atlas-PrivateEndpoint' ] ]
+#      Type: RESOURCE
+#      TypeName: MongoDB::Atlas::PrivateEndpoint
+#      ExecutionRoleArn: !GetAtt MongoDBPrivateEndpointExecutionRole.Arn
+#  ActivateNetworkContainerType:
+#    Type: AWS::CloudFormation::TypeActivation
+#    Properties:
+#      PublicTypeArn: !Join [ "", [ 'arn:aws:cloudformation:',!Ref "Region",'::type/resource/bb989456c78c398a858fef18f2ca1bfc1fbba082/MongoDB-Atlas-NetworkContainer' ] ]
+#      Type: RESOURCE
+#      TypeName: MongoDB::Atlas::NetworkContainer
+#      ExecutionRoleArn: !GetAtt MongoDBCustomResourceExecutionRole.Arn
+  MongoDBCustomResourceExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      MaxSessionDuration: 8400
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - cloudformation.amazonaws.com
+                - resources.cloudformation.amazonaws.com
+                - lambda.amazonaws.com
+            Action: sts:AssumeRole
+      Path: "/"
+      Policies:
+        - PolicyName: ResourceTypePolicy
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "secretsmanager:CreateSecret"
+                  - "secretsmanager:CreateSecretInput"
+                  - "secretsmanager:DescribeSecret"
+                  - "secretsmanager:GetSecretValue"
+                  - "secretsmanager:PutSecretValue"
+                  - "secretsmanager:UpdateSecretVersionStage"
+                  - "ec2:CreateVpcEndpoint"
+                  - "ec2:DeleteVpcEndpoints"
+                  - "cloudformation:CreateResource"
+                  - "cloudformation:DeleteResource"
+                  - "cloudformation:GetResource"
+                  - "cloudformation:GetResourceRequestStatus"
+                  - "cloudformation:ListResources"
+                  - "cloudformation:UpdateResource"
+                  - "iam:AttachRolePolicy"
+                  - "iam:CreateRole"
+                  - "iam:DeleteRole"
+                  - "iam:GetRole"
+                  - "iam:GetRolePolicy"
+                  - "iam:ListAttachedRolePolicies"
+                  - "iam:ListRolePolicies"
+                  - "iam:PutRolePolicy"
+                Resource: '*'
```

Comment:
is this needed? I would expect the user to pass the arn of the role as input param